assets/src/design-system/theme/helpers/outline.js

```diff
@@ -45,8 +45,6 @@ export const focusableOutlineCSS = (colorOrProps, background) => {
       ? colorOrProps
       : colorOrProps?.theme?.colors?.border?.focus;
   return css`
-    border: 2px solid transparent;
```

Comment:
this resurfaced tonight when i rebased. @samwhale can you just triple check focus updates are still present in main?

Reply:
good catch - this is in main too and should be removed.